internal/lock_test.go

```diff
@@ -115,7 +115,7 @@ func TestDownload(t *testing.T) {
 	lock, err := NewLock(path, false)
 	assert.Nil(t, err)
 	dir := test.TmpDir(t)
-	err = lock.Download(dir, []string{}, []string{}, perm)
+	err = lock.Download(dir, []string{}, []string{}, perm, true)
```

Comment:
This last param should be false: here we're testing the lock code. We don't want to exercise the status code.
But we do want to add a full download with status enabled in status_test.go